airbyte-workers/src/main/java/io/airbyte/workers/WorkerApp.java

```diff
@@ -4,18 +4,21 @@
 
 package io.airbyte.workers;
 
+import com.auth0.jwt.JWT;
```

Comment:
@jdpgrailsdev are the changes in this class in line with what we would want to do with micronaut? specifically the uses of statics.

Reply:
discussion continued in child PR here: https://github.com/airbytehq/airbyte/pull/15904/files#r953197010